_data/providers.json

```diff
@@ -6,6 +6,11 @@
     "java_ip": "Make sure your remote address is 'auto', uncomment `bedrock address`, and change `0.0.0.0` to your Java server's IP."
   },
   "built_in": [
+    {
+      "name": "GameProHost",
+      "url": "https://gameprohost.com/",
+      "description": "To install Geyser on GameProHost is very simple, you just need to install the plugin and set true in the port cloning "clone-remote-port: true" in the Geyser configuration. As an optional step, you can install the "Floodgate" plugin."
+    },
```

Comment:
It sounds like the `default` description template can be used instead - see the Physgun entry for an example

Reply:
Ready